docs/website/docs/reference/performance_snippets/performance-snippets.py

```diff
@@ -179,9 +179,7 @@ async def _run_async():
                 loop.run_in_executor(executor, _run_pipeline, pipeline_1, async_table),
                 loop.run_in_executor(executor, _run_pipeline, pipeline_2, defer_table),
             )
-        # result contains two LoadInfo instances
-        results[0].raise_on_failed_jobs()
-        results[1].raise_on_failed_jobs()
+        # results contains two LoadInfo instances
```

Comment:
Without the references into the `results` list this example is less instructive. I am not sure whether to:
1. leave only the comment
2. leave the calls to `results[n].raise_on_failed_jobs()`
3. call some other method/property, e.g. print(results[0]) or `results[0].asdict()`